go/vt/vtgate/evalengine/eval_result.go

```diff
@@ -339,16 +371,16 @@ func (er *EvalResult) makeTextualAndConvert(collation collations.ID) bool {
 }
 
 func (er *EvalResult) truncate(size int) {
```

Comment:
nit but IMO the function should be called ~ `enforceSize` since we truncate or pad as needed.

Reply:
:+1: will refactor